src/main/kotlin/com/exactpro/th2/codec/ValidateException.kt

```diff
@@ -0,0 +1,25 @@
+package com.exactpro.th2.codec
+
+class ValidateException : CodecException {
+
+    var title: String? = null
```

Comment:
I think we can also add some details to the message and we will add them as a _message bean_ to the event body